commit a1d969d47a5d2e49a7c3a0a0b6c7e6ed9fd46622
Author: eumagga0x2a <eumagga0x2a@users.noreply.github.com>
Date:   Sun Jul 5 23:21:25 2020 +0200

    [ass] Use the right header when building with system libass

diff --git a/avidemux_plugins/ADM_videoFilters6/ass/CMakeLists.txt b/avidemux_plugins/ADM_videoFilters6/ass/CMakeLists.txt
index af432d455..c2f31ef2b 100644
--- a/avidemux_plugins/ADM_videoFilters6/ass/CMakeLists.txt
+++ b/avidemux_plugins/ADM_videoFilters6/ass/CMakeLists.txt
@@ -16,6 +16,7 @@ IF ((USE_FREETYPE AND USE_FRIBIDI) OR MSVC)
             MESSAGE(STATUS "Checking libass external lib ...")
             PKG_CHECK_MODULES(LIBASS libass)
             IF (LIBASS_FOUND)
+                ADD_DEFINITIONS("-DUSE_EXTERNAL_LIBASS")
                 ADD_DEFINITIONS(${LIBASS_CFLAGS})
                 SET(LIBASS_LIBRARIES ${LIBASS_LDFLAGS})
                 MESSAGE(STATUS "   found, lib is ${LIBASS_LIBRARIES}")
